packages/extension/test/integration/background_spec.js

```diff
@@ -274,49 +274,6 @@ describe('app/background', () => {
   })
 
   context('add header to aut iframe requests', () => {
-    it('does not add header if it is the top frame', async function () {
```

Comment:
since we filter the type in the webrequest API vs programmatically we can no longer unit test these

Reply:
Actually scratch that. There was a bug with this removal where we needed to check for the parent frame, which was setting the AUT url to things that were not the AUT url. This is fixed in [99964fb](https://github.com/cypress-io/cypress/pull/26623/commits/99964fbfbda50370fbe7fc111482a081f24a9e47)